src/psycopmlutils/synth_data_generator/synth_col_generators.py

```diff
@@ -10,7 +11,7 @@
 
 
 def create_outcome_values(
-    df: pd.DataFrame,
+    df: pd.DataFrame,  # pylint: disable=redefined-outer-name
```

Comment:
er der et outer name some hedder df?

Reply:
fra if __name__ scope?

Reply:
Yeps – det er der det opstår. Har ikke noge oplagt løsning her, mega åben for forslag 👍

Reply:
Jeg vil slå det fra for pylint (flake8 tjekker det og gør det bedre - der er nogle edge cases med pytest fixtures den ikke fanger)

Reply:
Good point, done!